lib/util.js

```diff
@@ -76,7 +76,8 @@ const inspectDefaultOptions = Object.seal({
   showProxy: false,
   maxArrayLength: 100,
   breakLength: 60,
-  compact: true
+  compact: true,
+  minComplexity: 1e5
```

Comment:
Can you add a trailing comma?  Ditto on line 309.

Reply:
I would like to keep this as is currently. I agree that it would be better to change this but it does not seem like [others agree](https://github.com/nodejs/node/pull/20703). So I would only want to change this if https://github.com/eslint/eslint/issues/10350 gets implemented and we use that rule.

Reply:
Okay, fair enough.

<sup>I've been surreptitiously sneaking in commas though.  Don't tell.</sup>